content/_layouts/frame.html.haml

```diff
@@ -124,3 +124,9 @@
           $body.removeClass("no-outline");
         })
       })
+    :javascript
```

Comment:
Please create a dedicated javascript file for dedicated page logic, instead of adding it to a top level page.